l10n_it_ricevute_bancarie/models/account.py

```diff
@@ -412,6 +415,21 @@ def action_riba_issue(self):
                     )
                     % (line.name, line.partner_id.name)
                 )
+            if (
+                line.distinta_line_ids
+                and line.distinta_line_ids.riba_line_id.state == "accredited"
```

Comment:
Questo dovrebbe essere sufficiente
```suggestion
                line.distinta_line_ids.riba_line_id.state == "accredited"
```